runbotics/runbotics-orchestrator-ui/src/pages/_app.tsx

```diff
@@ -40,6 +41,12 @@ moment.locale(DEFAULT_LANG);
 // Client-side cache, shared for the whole session of the user in the browser.
 const clientSideEmotionCache = createEmotionCache();
 
+(async function initializeBlogCache() {
+    if (!isCached('en') || !isCached('pl')) {
+        await recreateCache();
+    }
+})();
```

Comment:
are you sure it runs at app start?

Reply:
to me its more like if someone will visit any page of the app

Reply:
I tested this solution in development and production mode. In development mode, the compilation is executed only when the user enters the page, so this logic is run only when the user visits the page. In production mode, the application runs based on already compiled code, so the logic here is executed after the application is launched without the user having to visit the page

Reply:
I can provide an additional condition to clarify that this logic is intended for `production` purposes only, eg:
`if (process.env.NODE_ENV === 'production' && (!isCached('en') || !isCached('pl')))`